components/dashboard/src/workspaces/StartWorkspaceModal.tsx

```diff
@@ -5,6 +5,7 @@
  */
 
 import { useEffect, useState } from "react";
+import { gitpodHostUrl } from "../service/service";
```

Comment:
<details>
<summary>:warning: <strong>This code is using `"` double-quotes in `require` or `import` statement.</strong> (click for details)</summary>

A possible improvement would be to follow best conventions of JS and use `'` single-quote to declare path in `require` statements.  
  
This change would make the code clearer and consistent.

**[More details](https://confluence.devfactory.com/x/DXFnFQ)**

## The improvement suggestion in this case
```tsx  
import { gitpodHostUrl } from '../service/service';
```  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="7eb22257-1219-4d71-b114-c99851b379c1" data-finding-id="359100685" href="none" codereview-uuid-reserved-tag data-business-id="3d94a8d2-7431-4dbe-bce7-80b87bb68d39" data-analyzer="js" />)